packages/mineral-ui-tokens/scripts/build-tokens.js

```diff
@@ -11,6 +11,7 @@ const colorIndex = require('../tokens/palette');
 const MINERAL_TOKENS_DIR = path.join(__dirname, '../tokens');
 const MINERAL_TOKENS_COLORS_DIR = path.join(MINERAL_TOKENS_DIR, '/palette');
 const SRC_DIR = path.join(__dirname, '../src');
+const THEMES_DIR = path.join(__dirname, '../../../src/library/themes');
```

Comment:
minor aside: The naming convention on these DIR constants could prolly be improved.

Reply:
Open to suggestions.

Reply:
Generate internal-only files into `library/themes/generated` and `website/../generated`.